ClosedXML_Tests/Excel/Worksheets/XLWorksheetTests.cs

```diff
@@ -815,5 +815,20 @@ public void RenameWorksheetChangesAbsoluteReferencesInFormulae()
                 Assert.AreEqual("Renamed!A1 * 3", ws1.Cell("A2").FormulaA1);
             }
         }
+
```

Comment:
I think we need a test for a global named range, where the range is first deleted, then the worksheet.

Reply:
TBD

Reply:
Is this related to #686 ?

Reply:
Not exactly. I created #880 for this.